### DISCUSSION TITLE
Trailing slash issue with netlify

### DISCUSSION THREAD

**Discussion by rrousselGit** [facebook/docusaurus#8524] [Q&A]
Hello!

This is related to https://github.com/facebook/docusaurus/issues/3372 and https://github.com/slorber/trailing-slash-guide/blob/main/docs/Hosting-Providers.md

I am surprisingly in a situation where Netlify doesn't behave like the trailing slash guide seems to argue it behaves.

I'm using the default trailingSlash config, yet somehow a trailing slash appears at the end of file links.
So instead of /docs/getting_started, I am redirected to /docs/getting_started/ (breaking relative links in the process). 

Any thoughts on what could be the issue?  Or maybe the guide is out of date?

**Comment by slorber** [comment]
It is possible that Netlify changed something and the guide is out of date, but I doubt it changed because it would be a quite risky change for Netlify (unless they apply it to new deployments only).

To be helpful I'd need an URL to reproduce, the trailing slash config, and a screenshot of the Netlify config.

Also make sure you have read this warning and understand the confusing checkbox weirdness happening on the Netlify settings UI:

https://docusaurus.io/docs/deployment#deploying-to-netlify

<img width="985" alt="CleanShot 2023-01-18 at 18 49 55@2x" src="https://user-images.githubusercontent.com/749374/213256824-bbbb9ef2-fcda-4ed1-bcce-ea3241c549e6.png">

What we use for Docusaurus in production:

<img width="890" alt="CleanShot 2023-01-18 at 18 52 57@2x" src="https://user-images.githubusercontent.com/749374/213257387-7fa1dc09-7d9d-49e8-ab61-989e9322dd35.png">

What doesn't work well last time I checked:

<img width="598" alt="CleanShot 2023-01-18 at 18 53 28@2x" src="https://user-images.githubusercontent.com/749374/213257450-570ba738-11ae-416a-8a57-c63f09dd81a4.png">

(I know, this is weird... reported this bug multiple times, not sure it's fixed already...)
